core/src/avm2/traits.rs

```diff
@@ -222,6 +335,7 @@ impl<'gc> Trait<'gc> {
                     disp_id: *disp_id,
                     method: unit.load_method(*method, false, activation)?,
                 },
+                metadata: metadata_from_abc_trait(activation, unit, abc_trait.metadata.clone())?,
```

Comment:
I thought we wanted these to be loaded lazily (at `describeType()` time) from the TU, rather than eagerly? Most metadata is not used so we'd prefer to just store the index and forget about it.

Reply:
This will be resolved to None very quickly if there's no metadata